headlamp-plugin/Makefile

```
@@ -0,0 +1,63 @@
+		$(IMAGE_REGISTRY)/$(IMAGE_REPO):$(IMAGE_TAG_NO_ARCH)-arm64
+
+.PHONY: image-manifest-annotate
+image-manifest-annotate:
+	$(eval IMAGE_TAG_NO_ARCH := $(patsubst %-amd64,%,$(patsubst %-arm64,%,$(IMAGE_TAG))))
+	docker manifest annotate $(IMAGE_REGISTRY)/$(IMAGE_REPO):$(IMAGE_TAG_NO_ARCH) \
+		$(IMAGE_REGISTRY)/$(IMAGE_REPO):$(IMAGE_TAG_NO_ARCH)-amd64 --arch amd64
+	docker manifest annotate $(IMAGE_REGISTRY)/$(IMAGE_REPO):$(IMAGE_TAG_NO_ARCH) \
+		$(IMAGE_REGISTRY)/$(IMAGE_REPO):$(IMAGE_TAG_NO_ARCH)-arm64 --arch arm64
+
+.PHONY: image-manifest-push
+image-manifest-push:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)